bean-validation/3.0/_index.md

```diff
@@ -1,25 +1,23 @@
 ---
-title: "Jakarta Bean Validation 3.0 (Milestone)"
-date: 2020-06-09
+title: "Jakarta Bean Validation 3.0"
+date: 2020-07-07
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta Bean Validation defines a metadata model and API for JavaBean and method validation.
 
 * [Jakarta Bean Validation 3.0 Release Record](https://projects.eclipse.org/projects/ee4j.bean-validation/releases/3.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Bean Validation 3.0 Specification Document](./bean-validation-specification-3.0.0-M1.pdf) (PDF)
-* [Jakarta Bean Validation 3.0 Specification Document](./bean-validation-specification-3.0.0-M1.html) (HTML)
+* [Jakarta Bean Validation 3.0 Specification Document](bean-validation-specification-3.0.pdf) (PDF)
+* [Jakarta Bean Validation 3.0 Specification Document](bean-validation-specification-3.0.html) (HTML)
 * [Jakarta Bean Validation 3.0 Javadoc](./apidocs)
-* [Jakarta Bean Validation 3.0 TCK](http://download.eclipse.org/ee4j/bean-validation/3.0/beanvalidation-tck-dist-3.0.0-M2.zip)([sig](),[sha](),[pub]())
-  * TCK Summary Results [Hibernate Validator 7.0.0.Alpha3](./TCK-summary.html)
+* [Jakarta Bean Validation 3.0 TCK](http://download.eclipse.org/ee4j/bean-validation/3.0/beanvalidation-tck-dist-3.0.0.zip)([sig](),[sha](bb0c85c6ab78786b023931cc407e00bbcd96fc26c0d8b56a00c28bc1665c7f01),[pub]())
 
 * Maven coordinates
-  * [jakarta.bean-validation:jakarta.bean-validation-api:jar:3.0.0-M1](https://repo1.maven.org/maven2/jakarta/validation/jakarta.validation-api/3.0.0-M1/jakarta.validation-api-3.0.0-M1.jar)
-
+  * [jakarta.bean-validation:jakarta.bean-validation-api:jar:3.0.0](https://repo1.maven.org/maven2/jakarta/validation/jakarta.validation-api/3.0.0/jakarta.validation-api-3.0.0.jar)
```

Comment:
FYI, the API jar files should not be promoted until the PR has passed ballot.  The Spec PR review process may find something that would need to be corrected in the API or the Javadoc, so this should be left in staging until the ballot has passed.